Spigot-API-Patches/0231-added-PlayerChunkReceiveEvent.patch

```diff
@@ -0,0 +1,54 @@
+From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
+From: ysl3000 <yannicklamprecht@live.de>
+Date: Mon, 5 Oct 2020 21:24:45 +0200
+Subject: [PATCH] added PlayerChunkReceiveEvent
+
+
+diff --git a/src/main/java/org/bukkit/event/packet/PlayerChunkReceiveEvent.java b/src/main/java/org/bukkit/event/packet/PlayerChunkReceiveEvent.java
+new file mode 100644
+index 0000000000000000000000000000000000000000..e23bb5de0cd72a8181f7c92a029b7715732a6aa9
+--- /dev/null
++++ b/src/main/java/org/bukkit/event/packet/PlayerChunkReceiveEvent.java
+@@ -0,0 +1,42 @@
++package org.bukkit.event.packet;
++
```

Comment:
rm ws

Reply:
Don't remove whitespace where it falls in line with formatting of literally every other Java file in this project.